### ISSUE TITLE
setup-minikube does not install crane

### ISSUE THREAD

**Issue by sunnyyip** [buildsec/frsca#362]
# Bug Report

setup-minikube does not install crane

## Current Behavior

The setup-minikube make step does not install crane which is required to run through the examples.

## Expected Behavior

Install all tools necessary to complete the examples

## Steps to Reproduce

1.
2.
3.

## Your Environment

```txt
(replace the example below with the script output)
Last commit:
  d6d7647 Alternate approach for internal minikube OCI registry  (HEAD -> github-templates, upstream/main, upstream/HEAD, main)
Kubernetes:
  Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.4", GitCommit:"b695d79d4f967c403a96986f1750a35eb75e75f1", GitTreeState:"clean", BuildDate:"2021-11-17T15:41:42Z", GoVersion:"go1.16.10", Compiler:"gc", Platform:"darwin/arm64"}
Tekton:
  Client version: 0.21.0
```

**Comment by developer-guy** [comment]
I can take care of this one @sudo-bmitch, you can assign it to me

**Comment by sudo-bmitch** [comment]
I think this is being resolved by #364
